cam/segmenter.go

```diff
@@ -83,7 +85,7 @@ func newSegmenter(
 		return nil, fmt.Errorf("failed to copy codec parameters %s", ffmpegError(ret))
 	}
 
-	segmentLengthCStr := C.CString(fmt.Sprintf("%d", clipLength))
+	segmentLengthCStr := C.CString(strconv.Itoa(clipLength))
```

Comment:
This is tested to prove it was equivalent to the behaviour before?

Reply:
Yep, segment clip lengths are the correct size.